src/SprykerEco/Zed/UnzerGui/Communication/Plugin/MerchantGui/MerchantUnzerFormExpanderPlugin.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerGui\Communication\Plugin\MerchantGui;
+
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+use Spryker\Zed\MerchantGuiExtension\Dependency\Plugin\MerchantFormExpanderPluginInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @method \SprykerEco\Zed\UnzerGui\Communication\UnzerGuiCommunicationFactory getFactory()
+ */
+class MerchantUnzerFormExpanderPlugin extends AbstractPlugin implements MerchantFormExpanderPluginInterface
```

Comment:
Plugin names should have a suffix of plugin interface name, 
e. g. `MerchantFormExpanderPluginInterface ` → `[Unzer]MerchantFormExpanderPlugin`.

Reply:
Fixed